### PR TITLE
Updated token number issue #413

### PR DESCRIPTION
**Pull request by rahulpatel596**
Updated token number. 
Also added media queries for small sized devices.

### CODE REVIEW

File: simplq/src/styles/statusPage.module.scss [hunk 1]
```diff
@@ -103,17 +103,79 @@
...
 .token-number {
   z-index: 10;
-  padding: 5%;
-  p {
+  width:225px;
+  span {
     @include center-horizontally();
     @include text-color();
-    font-size: 1rem;
+    // font-size: 1rem;
```
Comment (daltonfury42) [inline simplq/src/styles/statusPage.module.scss:113]: remove

**Review by daltonfury42 — COMMENTED** [review]
I will review this weekend, unless @maaverik reviews and merges first.

Thanks for the PR, this is looking good on a quick look

File: simplq/src/styles/statusPage.module.scss (hunk 1 above)
Comment (rahulpatel596) [reply simplq/src/styles/statusPage.module.scss]: removed comment. It should be font-size: 1 rem; now

File: simplq/src/styles/statusPage.module.scss [hunk 2]
```diff
@@ -105,15 +105,75 @@
...
+  }
+
+  //For Mobile and small sized devices.
+  @media only screen and (max-width: 600px) {
+    margin-left: 35px;
+    .hanging-threads {
+      height: 55px;
+      margin-top: -35px;
+    }
   }
```
Comment (daltonfury42) [inline simplq/src/styles/statusPage.module.scss:178]: Hey, sorry for the late review. I decided to take last weekend off, and was focusing on non-tech stuff.

Can you convert from `px` to `rem` wherever possible?

Comment (rahulpatel596) [reply simplq/src/styles/statusPage.module.scss]: Made changes, good to go 👍

**Review by daltonfury42 — APPROVED** [review]

**Comment by daltonfury42** [comment]
This closes #413 

Thanks, @rahulpatel596 

Your changes can be seen on https://dev.simplq.me/ and we will gradually update the main site.

**Pull request merged**